src/stretchy.js

```diff
@@ -154,98 +153,108 @@ const groupLength = function(arg: ParseNode): number {
     }
 };
 
-const svgSpan = function(group: ParseNode, options: Options): span {
+const svgSpan = function(group: ParseNode, options: Options): domTree.span {
     // Create a span with inline SVG for the element.
-    const label = group.value.label.substr(1);
-    let attributes = [];
-    let height;
-    let viewBoxWidth = 400000;  // default
-    let viewBoxHeight = 0;
-    let minWidth = 0;
-    let path;
-    let paths;
-    let pathName;
-    let svgNode;
-    let span;
```

Comment:
I made all these variables as local as possible since it was getting difficult for me to otherwise reason about their area of influence of any assignment to any of these variables.